lib/ffi/ogr/api.rb

```diff
@@ -13,253 +16,18 @@ module API
       # Typedefs
       # -----------------------------------------------------------------------
       # TODO: Use the typedefs for creating pointers throughout the app.
-      typedef :pointer, :OGRGeometryH
-      typedef :pointer, :OGRFieldDefnH
       typedef :pointer, :OGRFeatureDefnH
       typedef :pointer, :OGRFeatureH
       typedef :pointer, :OGRStyleTableH
-      typedef :pointer, :OGRGeomFieldDefnH
       typedef :pointer, :OGRLayerH
       typedef :pointer, :OGRDataSourceH
       typedef :pointer, :OGRSFDriverH
       typedef :pointer, :OGRStyleMgrH
       typedef :pointer, :OGRStyleToolH
 
-      # -----------------------------------------------------------------------
```

Comment:
I moved all of these out to child files simply so I could gradually add type-checking for the functions in this module